sencha-workspace/SlateTasksTeacher/app/view/StudentsGrid.js

```diff
@@ -2,312 +2,233 @@
+            '</table>'
+        ],
+        expanderBodyTpl: [
+            '<table class="jarvus-aggregrid-expander-table">',
+                '<tbody>',
+                    '<tpl for="subRows">',
+                        '<tr class="jarvus-aggregrid-subrow" data-subrow-id="{ID}">',
+                            '<tpl for="columns">',
+                                '<td class="jarvus-aggregrid-cell {cls}" data-column-id="{ID}">{Title}</td>',
                             '</tpl>',
                         '</tr>',
-                    '</thead>',
-
-                    '<tbody>',
-                        '<tpl for="rows">',
-                            '<tr class="slate-studentsgrid-row">',
-                                '<tpl for="students">',
-                                    '<td class="slate-studentsgrid-cell {cls}">{text}</td>',
-                                '</tpl>',
-                            '</tr>',
+                    '</tpl>',
+                '</tbody>',
+            '</table>'
+        ]
+    },
 
-                            // expander infrastructure
-                            '<tr class="slate-studentsgrid-expander">',
-                                '<td class="slate-studentsgrid-expander-cell" colspan="{[ studentsCount ]}">',
-                                    '<div class="slate-studentsgrid-expander-ct">',
-                                        '<table class="slate-studentsgrid-expander-table">',
-                                            '<tbody>',
-                                            //
-
-                                                '<tpl for="rows">',
-                                                    '<tr class="slate-studentsgrid-subrow">',
-                                                        '<tpl for="students">',
-                                                            '<td class="slate-studentsgrid-cell {cls}">{text}</td>',
-                                                        '</tpl>',
-                                                    '</tr>',
-                                                '</tpl>',
-
-                                            //
-                                            '</tbody>',
-                                        '</table>',
-                                    '</div>',
-                                '</td>',
-                            '</tr>',
-                            //
-                        '</tpl>',
-                    '</tbody>',
-                '</table>',
-            '</div>',
-        '</div>'
-    ],
+    initComponent: function() {
+        var me = this;
 
-    listeners: {
-        scope: 'this',
-        click: {
-            fn: 'onGridClick',
-            element: 'el',
-            delegate: '.slate-studentsgrid-row'
-        }
+        me.callParent(arguments);
+        me.setConfig({
+            cellRenderer: me.cellRenderFn,
+            subCellRenderer: me.cellRenderFn
```

Comment:
@themightychris Not necessary, just thought it would be better for them to use the same exact render function without duplicating the code.